timemachine/fe/interpolate.py

```diff
@@ -168,14 +168,14 @@ def log_linear_interpolation(src_params, dst_params, lamb, min_value):
 
 def pad(f, src_params, dst_params, lamb, lambda_min, lambda_max):
     """
-    Use the specified interpolation function in the interval [lambda_min, lambda_max], otherwise pin to the end-state
+    Use the specified interpolation function in the interval (lambda_min, lambda_max), otherwise pin to the end-state
```

Comment:
@mcwitt - does this seem reasonable? not sure what the original intent here was with the closed interval, open seems more appropriate (eg. interpolating over lambda_min=0, lambda_max=1 seems safer if we guarantee that the end-states are exactly as src_params, and dst_params, respectively).

Reply:
I think it's reasonable; there was no particular reason for the choice of closed interval initially (but it didn't matter then because `interpolate.pad` was never used directly to do parameter interpolation; it was wrapped with `handle_ring_opening_closing`, which special-cased lambda=0 and 1 to return the end-state values).

Reply:
thanks! resolving this